internal/app/wwctl/node/list/root.go

```diff
@@ -34,15 +33,14 @@ func GetCommand() *cobra.Command {
 			nodes, _ := nodeDB.FindAllNodes()
 			var node_names []string
 			for _, node := range nodes {
-				node_names = append(node_names, node.Id.Get())
+				node_names = append(node_names, node.Id())
 			}
 			return node_names, cobra.ShellCompDirectiveNoFileComp
 		},
 	}
 	baseCmd.PersistentFlags().BoolVarP(&vars.showNet, "net", "n", false, "Show node network configurations")
 	baseCmd.PersistentFlags().BoolVarP(&vars.showIpmi, "ipmi", "i", false, "Show node IPMI configurations")
 	baseCmd.PersistentFlags().BoolVarP(&vars.showAll, "all", "a", false, "Show all node configurations")
-	baseCmd.PersistentFlags().BoolVarP(&vars.showFullAll, "fullall", "A", false, "Show all node configurations inclusive empty entries")
```

Comment:
--fullall isn't possible to do any more, as NodeConf may not be initialized fully (pointer to Impi/Kernel may be nil). It even gets more complicated if you take Disks and partitions into account. I even don't know if this feature is used out there, so having a brittle complicated option if it may be even used is not worth the price.  That there is a --fullall for profiles is a bug.